Assets/Scenes/App Menu.unity

```diff
@@ -206,7 +206,7 @@ MonoBehaviour:
   m_outlineColor:
     serializedVersion: 2
     rgba: 4278190080
-  m_fontSize: 72
+  m_fontSize: 37.45
```

Comment:
It's a good idea for commits to have a single responsibility. This way instead of a commit doing A + B + C, we'd have 3 commits, each doing A, B or C. So if B has problems, A and C can still get through and be merged early.

Reply:
Yep, that was the plan, but somehow I managed to screw that and do twice the same change on two different branch... do you want me to rollback this?

Reply:
Yeah, let's keep it in a separate PR